scripts/build-on-ci.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
```

Comment:
This file seems completely unused, so I removed it rather than updating references of `npm` to `yarn`. Ref #938.